### ISSUE TITLE
Comms Mode for Max232 on CR6

### ISSUE THREAD

**Issue by violetteta8** [SacArea-BayDeltaProjects/HydroRDITemplate#28]
For CR6 loggers using ComC1 or ComC3 with a Max232, we need to use TTL logic instead of RS232. Need to update the SerialOpen commands FROM:
If UseComms(zero)ForSonde=True Then 
SerialOpen(WqSerialPort,WqBaud,19,0,2000,0)
(Change variable name from UseCommsZeroForSonde to UseCommsOneForSonde for consistency's sake).
TO:
If UseComms(zero)ForSonde = True Then
SerialOpen(WqSerialPort,WqBaud,19,0,2000,1)

Need to investigate ADCP port as well.

**Comment by USGSminc** [comment]
Made pull request change WQ data type and variables only. We can address ADVM later.

**Comment by violetteta8** [comment]
Merged WQ pull request.

**Comment by violetteta8** [comment]
CommsModeOneForSonde didn't work at Shag. With no Max232 I had to set that value to False to get data to come in. Haven't had a chance to look at it. But we need to address it.

**Comment by USGSminc** [comment]
Sorry, I completely forgot that Shag didn't have a max 232 for the sonde. Is that what you meant by the commsModeOneForSonde not working? Or was there another problem? I changed the com setting for the max 232 from 0 to 1 to make the max 232 work. Did that make it so it didn't work when there was no max 232 connected? Otherwise I think the only thing I did was chance the name from zero to one. I didn't realize that the two were separate. If that is the case we just need to recreate com mode zero for when it is only in serial mode?
